test/acceptance/behave/components/ngsiv2/subscriptions/notifications/notifications_metadata.feature

```diff
@@ -0,0 +1,493 @@
+    And properties to subscriptions
+      | parameter             | value                           |
+      | subject_idPattern     | .*                              |
+      | condition_attrs       | without condition field         |
+      | notification_http_url | http://replace_host:1045/notify |
+    And create a new subscription
+    And verify that receive a "Created" http code
+    # These properties below are used in entity request
+    And properties to entities
+      | parameter         | value       |
+      | entities_type     | random=3    |
+      | entities_id       | room1       |
+      | attributes_number | 3           |
+      | attributes_name   | temperature |
+      | attributes_value  | random=5    |
+      | attributes_type   | celsius     |
+      | metadatas_number  | 2           |
+      | metadatas_name    | very_hot    |
+      | metadatas_type    | alarm       |
+      | metadatas_value   | hot         |
+    When create entity group with "1" entities in "normalized" mode
+    And verify that receive several "Created" http code
+    Then get notification sent to listener
+    And verify metadata in notification with "*"
+
+  @metadata_create_entity  @metadata_create_entity.row<row.id>
+  Scenario Outline:  send a notification using NGSI v2 with special metadata when an entity is created
+    Given  a definition of headers
+      | parameter          | value                       |
+      | Fiware-Service     | test_notif_metadata_created |
+      | Fiware-ServicePath | /test                       |
+      | Content-Type       | application/json            |
+    # These properties below are used in subscriptions request
+    And properties to subscriptions
+      | parameter             | value                           |
+      | subject_idPattern     | .*                              |
+      | condition_attrs       | without condition field         |
+      | notification_http_url | http://replace_host:1045/notify |
+      | notification_metadata | <metadata>                      |
+    And create a new subscription
+    And verify that receive a "Created" http code
+    # These properties below are used in entity request
+    And properties to entities
+      | parameter         | value       |
+      | entities_type     | random=3    |
+      | entities_id       | room2       |
+      | attributes_number | 3           |
+      | attributes_name   | temperature |
+      | attributes_value  | random=5    |
+      | attributes_type   | celsius     |
+      | metadatas_number  | 2           |
+      | metadatas_name    | very_hot    |
+      | metadatas_type    | alarm       |
+      | metadatas_value   | hot         |
+    When create entity group with "1" entities in "normalized" mode
+    And verify that receive several "Created" http code
+    Then get notification sent to listener
+    And verify metadata in notification with "<metadata>"
+    Examples:
```

Comment:
I take the `previousValue` and the `actionType` from previous steps (create and update request), and all tests are implemented in the notification_steps.py and the NGSI_v2.py libraries. I push these libraries in another PR. And another library in commom repository to create requests [(cb_v2_utils.py).](https://github.com/telefonicaid/iotqatools/blob/develop/iotqatools/cb_v2_utils.py)